backend-stack-bubbling/app.py

```diff
@@ -572,6 +580,47 @@ def post():
                 "is_upvote": info["is_upvote"]
             }), 200)
 
+class DeclareBestAnswer(Resource):
+    @staticmethod
+    @jwt_required()
+    def post():
+        info = BestAnswerInfo.parse_args()
+        identity = get_jwt_identity()
+        responseMessage = ""
+        currentUser = UserCollection.find_one(
+            {
+                "email": identity["email"]
+            })
+        questionID = uuid.UUID(info["question_id"])
+        answerID = uuid.UUID(info["answer_id"])
+        if currentUser is not None:
+            answer = QuestionCollection.find_one(
+                {
+                    "_id": questionID,
+                    "answers.is_best_answer": True
+                })
+            if answer is None:
+                QuestionCollection.update(
+                {
+                    "_id" : questionID,
+                    "answers._id": answerID
+                },
+                {
+                    "$set":  
+                    {
+                        "answers.$.is_best_answer": True
+                    }
+                })
+                responseMessage = "Best Answer Declared!"
+                return make_response(jsonify(responseMessage), 201)
+            else:
+                responseMessage = "Best Answer is already declared!"
+                return make_response(jsonify(responseMessage), 204)
```

Comment:
Be aware that the return message here is different than other function,
it is of form {"responseMessage"},
and not of form {"message":"responseMessage"}

Reply:
I just uploaded a fix for this. Now the POST request returns the message in the format you requested.